mirroroperator/registrymirror.py

```diff
@@ -43,6 +43,7 @@ def __init__(self, event_type, namespace, hostess_docker_registry,
 
         self.ss_ds_labels = kwargs["ss_ds_labels"] or ""
         self.ss_ds_template_labels = kwargs["ss_ds_template_labels"] or ""
+        self.ss_ds_tolerations = kwargs["ss_ds_tolerations"] or ""
```

Comment:
should this not default to an empty array?

Reply:
Maybe None is better?